charts/pega/README.md

```diff
@@ -1014,10 +1014,12 @@ Parameter   | Description   | Default value
 ---         | ---           | ---
 `hazelcast.image` | Reference the `platform/clustering-service` Docker image that you downloaded and pushed to your Docker registry that your deployment can access. | `YOUR_HAZELCAST_IMAGE:TAG`
 `hazelcast.clusteringServiceImage` | Reference the `platform/clustering-service` Docker image that you downloaded and pushed to your Docker registry that your deployment can access. | `YOUR_CLUSTERING_SERVICE_IMAGE:TAG`
-`hazelcast.migratorImage` | Reference the `platform/clustering-service-kubectl` Docker image that you downloaded and pushed to your Docker registry that your deployment can access. | `YOUR_MIGRATION_JOB_IMAGE:TAG`
 `hazelcast.enabled` |  Set as true if client-server deployment of Pega Platform is required, otherwise false. Note: Set this value as false for Pega platform versions below 8.6, if not set the installation will fail | `true`
 `hazelcast.clusteringServiceEnabled` |  Set as true if client-server deployment of Pega Platform is required, otherwise false. Note: Set this value as false for Pega platform versions below 8.8, if not set the installation will fail | `false`
-`hazelcast.migration` |  Set as true after creating prallel cluster to migrate the data and client switch over to new Hazelcast service | `false`
+`hazelcast.migration.enabled` |  Set as true after creating parallel cluster to migrate the data and client switch over to new Hazelcast service | `false`
+`hazelcast.migration.migrationJobImage` | Reference the `platform/clustering-service-kubectl` Docker image that you downloaded and pushed to your Docker registry that your deployment can access. | `YOUR_MIGRATION_JOB_IMAGE:TAG`
+`hazelcast.migration.embeddedToCSMigration` |  Set as true while creating parallel cluster in case of embedded source system | `false`
+`hazelcast.migration.skipRestart` |  Set as true while removing old hazelcast cluster after migration to avoid restart | `false`
 `hazelcast.replicas` | Number of initial members to join the Hazelcast cluster | `3`
 `hazelcast.username` | UserName to be used in client-server Hazelcast model for authentication | `""`
 `hazelcast.password` | Password to be used in client-server Hazelcast model for authentication | `""`
```

Comment:
Configures the password to be used in a client-server Hazelcast model for authentication between the nodes in the Pega deployment and the nodes in the Hazelcast cluster. This parameter configures the password credential in Hazelcast cluster and your Pega nodes so authentication occurs automatically.

Reply:
@taz-pega-work , this is an existing content will do the modification in our upcoming documentation story.